pages/cart-page/cart.css

```diff
@@ -164,6 +147,7 @@ main{
 }
 
 .cart-transactions button {
+    margin: auto;
     margin-top: 8px;
```

Comment:
this can be combined (for eg: margin: 8px auto 0; - in case you don't need bottom margin)